mscore/CMakeLists.txt

```diff
@@ -475,7 +475,6 @@ if (MINGW)
       ${CROSS}/lib/portaudio.dll
       ${CROSS}/opt/bin/libeay32.dll
       ${CROSS}/opt/bin/ssleay32.dll
-      ${CROSSQT}/bin/Qt5CLucene.dll
```

Comment:
A comment furter up implies that build/Linux+BSD/portable/copy-libs needs to get adjusted too?